syntax/pb_test.go

```diff
@@ -0,0 +1,67 @@
+package syntax
```

Comment:
Nit: It feels like this test should be in `translate/pb/test`, alongside all the files it imports?

Reply:
As `AssertCodesEvalToSameValue` now is in package `syntax`, so moving this file to package `pb` will causing cycle import which is not allowed in go. I will raise a new PR to update after this one.